.github/workflows/ci.yml

```diff
@@ -37,13 +38,27 @@ jobs:
           PYTHON_CMD_PREFIX: ""
           NODE_CMD_PREFIX: ""
   test:
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
     runs-on: ubuntu-latest
     name: Tests install script
     env:
       BUILD_DIR: /home/runner/work/django-base-site/new_project/
       SECRET_KEY: secret-key-for-testing-only
       READ_DOT_ENV_FILE: on
-      DOCKER_COMPOSE_PREFIX: docker compose run -u root --rm --no-deps web
+      DOCKER_COMPOSE_PREFIX: docker compose run -u root -e DATABASE_URL --rm --no-deps web
```

Comment:
This is here to override the `.env` file that is created when it runs `./scripts/start_new_project` on line 68.